content/strategy-goals/strategy/batch-changes/index.md

```diff
@@ -59,82 +59,67 @@ We have the overarching goal to [level up our enterprise-ready features](../inde
 - Creating changesets locally takes too long to be practical for very large-scale changes.
 - Navigation in the UI when there are 100s of batch changes and 10,000s of changesets is cumbersome.
 - Batch Changes's permission model is minimalistic and insufficient for some large enterprises.
+- Supporting running batch changes in large monorepos
 
 To drive enterprise success and stickiness, we need to:
 
 - Deliver a great experience in creating changesets at **enterprise scale** (10,000s repositories), resulting in more users onboarding batch changes.
-- Make sure that iterating on batch changes has the **lowest cycle time possible**, and that the debugging experience is smooth.
+- Support enterprise requirements in terms of permissions and controls.
 - Enable our users to make sense of a large number of batch changes and changesets.
 
 ### Get changesets merged
 
 The key [success metric](../../../departments/engineering/teams/batch-changes/metrics.md) for batch changes is the number of changesets that get _merged_. In practice, once a user has applied a batch change, they need to spend time tracking and following up with downstream repository owners so that they merge the batch change's changesets. The larger the batch change, the more time-consuming this can be. We want to improve how easy it is for users to get large batch changes over the finish line, as measured by time-to-merge and batch changes merge rate.
 
-### Time-to-value
+### Time to value
 
-Batch Changes is a powerful product with a steep learning curve. The main friction point is that Batch Changes requires users to write **code rewrite steps** (the components of a batch spec that define a specific change to be executed), either using existing code rewrite tools (sed, comby, etc) or writing code from scratch. Most developers are unfamiliar with such tooling, and have a hard time onboarding to Batch Changes.
+Batch Changes is a powerful product with a steep learning curve. The two main friction points are:
```

Comment:
I agree with Chris here. This feedback has been shared more than once and it is a product issue we need to address.

Reply:
@danielmarquespt can you point to the source of this feedback? I remember some internal feedback, by a non-engineer. I don't remember feedback from users that are our primary target, although I agree this would be under-reported.

Reply:
You are right, the feedback that I recall is internal and from non-developers. However, I have a strong assumption (no data) that this is also experienced by our primary target, why:

There is nothing on the experience today that communicates confidence or safety _only_ to developers. This is not a matter of "those that are not technical didn't understand the process, so they are afraid". I think that developers simply have a better understanding of how git / permissions / repos work, or are able to revert anything bad that happens. They have more know-how, therefore they feel safe to solve anything that happens.

Non-technical audiences lack this "revert capability" therefore are extra cautious. That doesn't mean that the experience provides clarity for any of those audiences.

Again: all of this is my speculation, based on all my biases. So take for what it's worth.